app/code/Magento/Quote/Model/CustomerManagement.php

```diff
@@ -100,11 +105,47 @@ public function populateCustomerInfo(QuoteEntity $quote)
         }
     }
 
+    /**
+     * Validate 'CustomerAddressId' in quote for a newly created customer.
+     *
+     * @param QuoteEntity $quote
+     * @return void
+     */
+    public function validateCustomerAddressId($quote): void
```

Comment:
Can we make this method private and define argument types?
```suggestion
    private function validateCustomerAddressId(QuoteEntity $quote): void
```